src/pages/[platform]/build-a-backend/data/working-with-files/index.mdx

```diff
@@ -1364,28 +1343,17 @@ class SongViewModel: ObservableObject {
         }
 
         // Get the song record
-        let result = try await Amplify.API.mutate(request: .get(Song.self, byId: currentSong.id))
-        guard case .success(let queriedSong) = result else {
-            print("Failed with error: ", result)
-            return
-        }
-        guard var song = queriedSong else {
-            print("Song may have been deleted, no song by id: ", currentSong.id)
-            await setCurrentSong(nil)
-            return
-        }
-        guard song.coverArtKey != nil else {
-            print("There is no cover art key to remove image association")
+        var song = try await Amplify.API.mutate(request: .get(Song.self, byIdentifier: currentSong.id)).get()
+
+        guard song?.coverArtPath != nil else {
```

Comment:
guard on song and then guard on `song.coverArtPath != nil`, otherwise the print below "There is no cover art..." may be inaccurate since song could also be nil